utils/parserConfigs/index.js

```diff
@@ -1,12 +1,31 @@
 const path = require('path');
-const {  parseNumber, validUrl, validLogLevel } = require('./validators');
+const { mergeWithPriority } = require('../utils');
+const { parseNumber, validUrl, validLogLevel, validGlobalConfig } = require('./validators');
 
 const getConfigs = () => {
-  const configs = {
+  let configs = {
     features: path.join(__dirname, 'split.yml'),
     core: {},
   };
 
+  if (process.env.SPLIT_EVALUATOR_GLOBAL_CONFIG) {
```

Comment:
I think you can take advantage of Object.assign(origin, nullables, overrides) and get everything in one row